crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/reports/service/EmailNotificationService.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.justice.laa.crime.crowncourt.reports.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FileUtils;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class EmailNotificationService {
+
+    private static final String DATE = "date";
+    private static final String LINK_TO_FILE = "link_to_file";
+
+    private final NotificationClient client;
+
+    @Setter
+    @Value("${emailClient.notify.template-id}")
+    private String templateId;
+
+    @Setter
+    @Value("#{'${emailClient.notify.recipient}'.split(',')}")
+    private List<String> emailAddresses;
```

Comment:
What happens if there are no emails or only one?

Reply:
Another solution (maybe a bit overkill? or something we should/can include in LAA commons?) would be to create and register our own custom String-> List<String> converter like below:

```
@Component
public class StringToListConverter implements Converter<String, List<String>> {

  private static final String COMMA_SEPARATOR = ",";

  @Override
  public List<String> convert(@NotNull String commaSeparatedString) {

    if (StringUtils.isBlank(commaSeparatedString)) {
      return Collections.emptyList();
    }

    if (!commaSeparatedString.contains(COMMA_SEPARATOR)) {
      return Collections.singletonList(commaSeparatedString);
    }

    return Arrays.stream(commaSeparatedString.split(COMMA_SEPARATOR)).toList();
  }
}
```

and the config class:

```
@Configuration
public class WebMvcConfig implements WebMvcConfigurer {

  @Override
  public void addFormatters(FormatterRegistry registry) {
    registry.addConverter(new StringToListConverter());
  }
}
```

Reply:
It will return an empty list if there is no email ID. 
It will return a list with one email ID if there is an email ID.
But, there is no chance to empty list as we are configuring the email ID in our value yaml file for all our environments